src/layer.rs

```diff
@@ -1165,9 +1169,7 @@ where
             }
 
             // Assign end time, build and start span, drop span to export
```

Comment:
> // Assign end time

I think that the comment should have been updated too

Reply:
Oops! Want to send a PR?

Reply:
=> https://github.com/tokio-rs/tracing-opentelemetry/pull/148